examples/cisco/demo_ospf.pp

```diff
@@ -51,6 +51,7 @@
     bfd                      => true,
     default_metric           => '5',
     log_adjacency            => 'detail',
+    redistribute             => [['eigrp 1', 'rtmap_eigrp_1'], ['direct',  'rtmap_direct']]
```

Comment:
add a ',' to the end

Reply:
Thanks! I didn't even look at these, should have ran the check before pushing :-(